File: Python-Task-04/C/password_vault.c
#include <stdio.h>

int main() {
    FILE *fp;
    char website[50];
    char username[100];
    char password[50];
    char ch;

    printf("Enter Website Name: ");
    scanf("%s", website);

    printf("Enter Username: ");
    scanf("%s", username);

    printf("Enter Password: ");
    scanf("%s", password);

    fp = fopen("vault.txt", "a");

    fprintf(fp, "Website: %s\n", website);
    fprintf(fp, "Username: %s\n", username);
    fprintf(fp, "Password: %s\n", password);
    fprintf(fp, "------------------------------\n");

    fclose(fp);

    printf("\nRecord Saved Successfully\n");

    printf("\nSaved Records:\n\n");

    fp = fopen("vault.txt", "r");

    while((ch = fgetc(fp)) != EOF) {
        printf("%c", ch);
    }

    fclose(fp);

    return 0;
}

File: Python-Task-04/Python/password_vault.py
website = input("Enter Website Name: ")
username = input("Enter Username: ")
password = input("Enter Password: ")

with open("vault.txt", "a") as file:
    file.write(f"Website: {website}\n")
    file.write(f"Username: {username}\n")
    file.write(f"Password: {password}\n")
    file.write("-" * 30 + "\n")

print("\nRecord Saved Successfully\n")

with open("vault.txt", "r") as file:
    print(file.read())